lib/irb/command/base.rb

```diff
@@ -29,19 +33,13 @@ def help_message(help_message = nil)
 
         private
 
-        def string_literal?(args)
-          sexp = Ripper.sexp(args)
-          sexp && sexp.size == 2 && sexp.last&.first&.first == :string_literal
-        end
-
         def highlight(text)
           Color.colorize(text, [:BOLD, :BLUE])
         end
       end
 
-      def self.execute(irb_context, *opts, **kwargs, &block)
-        command = new(irb_context)
-        command.execute(*opts, **kwargs, &block)
+      def self.execute(irb_context, arg)
+        new(irb_context).execute(arg)
```

Comment:
Should we always `rstrip` command arg here? I can't think of any case that we'd want trailing spaces. I think it'd also avoid issues like #905

Reply:
But if we `rstrip` the arg here, then we don't need to call them manually in commands like `Break`?

Reply:
There is no need and no effect adding rstrip here. `arg` does not end with space, ensured in `CommandArgTest#test_arg`.

I added rstrip in Break#execute because:
```ruby
arg = "" # this arg does not end with " ", already stripped
pre_cmds = "continue #{arg}" #=> "continue " # this string might end with " " if arg is an empty string
pre_cmds = "continue #{arg}".rstrip #=> "continue"
```